src/alarms/notifications/EditNotification.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import MDSpinner from "react-md-spinner";
+
+import { NotificationForm } from "./NotificationForm";
+
+class EditNotificationModel extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            currentNotification: null
+        };
+    }
+
+    componentDidMount() {
+        const { match, alarmType } = this.props;
+
+        (async () => {
+            const url = (alarmType === "RASTERS") ? (
+                `/api/v4/rasteralarms/${match.params.id}/`
+            ) : (
+                `api/v4/timeseriesalarms/${match.params.id}/`
+            )
+
+            const currentNotification = await fetch(url, {
+                credentials: "same-origin"
+            }).then(response => response.json());
+
+            this.setState({ currentNotification })
+        })();
+    }
+
+    render() {
+        if (
+            this.state.currentNotification &&
+            this.props.organisations.isFetching === false
```

Comment:
Why does this depend on organisations.fetching?

Reply:
It doesn't depend on organisations. So I deleted it.